logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -27,8 +32,8 @@ def stop
       LogStash::Instrument::Collector.instance.delete_observer(self)
     end
 
-    def update(time, snapshot)
-      @logger.debug("Metrics input: received a new snapshot", :snapshot => snapshot, :event => snapshot.to_event) if @logger.debug?
+    def update(time_of_snapshot, snapshot)
```

Comment:
Actually we could create a Snapshot which basically wrap the metric and the time WDYT?